tests/test_main.rs

```diff
@@ -19,9 +19,60 @@ fn test_run_script() {
     Command::cargo_bin("pn")
         .unwrap()
         .current_dir(&temp_dir)
-        .arg("run")
-        .arg("test")
+        .args(["run", "test"])
         .assert()
         .success()
         .stdout("hello world\n");
 }
+
+#[test]
+fn test_workspace_root() {
+    let temp_dir = tempfile::tempdir().unwrap();
+    let temp_dir_path = temp_dir.path();
```

Comment:
No, I get an error

```
error[E0599]: no method named `join` found for struct `TempDir` in the current scope
  --> tests/test_main.rs:55:38
   |
55 |     let package_json_path = temp_dir.join("package.json");
   |                             ---------^^^^----------------
   |                             |        |
   |                             |        this is an associated function, not a method
   |                             help: use associated function syntax instead: `TempDir::join("package.json")`
```

Reply:
My mistake. I'm sorry. Regardless though, if you apply [this suggestion](https://github.com/pnpm/pn/pull/15#discussion_r1225104545), it will be rendered irrelevant.

Reply:
Wait, I'm wrong again. `TempDir` doesn't implement `Deref` at all. I will have to edit the other suggestion.